CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## UNRELEASED
 
+### Fixed
+
+-   Fix that two dialogs are shown when entering full-screen mode for the first time. (#875)
+
+## v6.9
+
 ### Added
 
 -   Now the winlibs release on Windows includes LLVM. If you use the `clangd` in this release as the C++ Language Server, `<bits/stdc++.h>` should be properly recognized. (#878)
```

Comment:
How did you merge this? I'm really worried about other files...

Reply:
I just merge from master. It seems that there isn't any conflict. Should I pick that PR(that solve aur) instead of merging master? I think it is not so different.

Reply:
Sorry, it was merged by, me :new_moon_with_face: https://github.com/cpeditor/cpeditor/commit/d510b07cd89ada72b40c99a1a25f6d26f840e484#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed